runtime/hookable/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ target_link_libraries(j9hookable
 		j9thr
 )
 
+get_target_property(hookable_exports j9hook_obj EXPORTED_SYMBOLS)
```

Comment:
A comment here would help - I think this is reading the exports from the OMR code but not 100% sure